Makefile

```
@@ -290,11 +294,15 @@ acsfleetctl:
 	GOOS="$(GOOS)" GOARCH="$(GOARCH)" CGO_ENABLED=0  $(GO) build $(GOARGS) -o acsfleetctl ./cmd/acsfleetctl
 .PHONY: acsfleetctl
 
-binary: fleet-manager fleetshard-sync probe acsfleetctl
+email-sender:
+	GOOS="$(GOOS)" GOARCH="$(GOARCH)" CGO_ENABLED=0  $(GO) build $(GOARGS) -o emailsender/bin/email-sender ./emailsender/cmd/app
+.PHONY: email-sender
```

I'd use the wording without the dash everywhere to stay consistent with naming. Could you change that throughout this file?

```suggestion
emailsender:
	GOOS="$(GOOS)" GOARCH="$(GOARCH)" CGO_ENABLED=0  $(GO) build $(GOARGS) -o emailsender/bin/emailsender ./emailsender/cmd/app
.PHONY: emailsender
```

---

We use `fleet-manager` or `fleetshard-sync` names as well in the makefile. But there is also `acsfleetctl` 😕